internals/cli/cmd_help.go

```diff
@@ -101,11 +102,11 @@ type manfixer struct {
 func (w *manfixer) Write(buf []byte) (int, error) {
 	if !w.done {
 		w.done = true
-		if bytes.HasPrefix(buf, []byte(".TH pebble 1 ")) {
+		if bytes.HasPrefix(buf, []byte(fmt.Sprintf(".TH %s 1 ", cmd.Personality.ProgramName))) {
 			// io.Writer.Write must not modify the buffer, even temporarily
-			n, _ := w.Buffer.Write(buf[:9])
+			n, _ := w.Buffer.Write(buf[:5+len(cmd.Personality.ProgramName)])
 			w.Buffer.Write([]byte{'8'})
-			m, err := w.Buffer.Write(buf[10:])
+			m, err := w.Buffer.Write(buf[6+len(cmd.Personality.ProgramName):])
```

Comment:
I'm not sure what's correct here, but it looks like these values will be different to what they were before. They were `buf[:9]` and `buf[10:]`, now they'll be `buf[:11]` and `buf[12:]`.

Reply:
I take your word for it that the test passes! But I am curious how the arithmetic works out? Can you explain why 10 before is equivalent to 6+len("pebble")==12 now?